packages/core/admin/server/controllers/transfer/runner.js

```diff
@@ -1,7 +1,9 @@
 'use strict';
 
-const { remote } = require('@strapi/data-transfer/lib/strapi');
-const { UnauthorizedError } = require('@strapi/utils/lib/errors');
+const { strapi } = require('@strapi/data-transfer');
```

Comment:
Sorry, I missed this before, but this part makes me a bit nervous since `strapi` is also added to the global namespace in most places. I would extract `remote` directly just to avoid the possibility of any weird behavior.